docs/Contributors_Guide/conda_recipe.rst

```diff
@@ -0,0 +1,153 @@
+.. _cg_conda_recipe:
+
+************
+Conda Recipe
+************
+
+Files used to create a Conda recipe can be found in **internal/scripts/conda**.
```

Comment:
@georgemccabe could you please add a sentence or two about when/whey a developer may want to build the conda environment locally? For example, to test adding to or modifying the conda build, add support for additional targets, or to debug a problem with an existing build.